### DISCUSSION TITLE
combined time and km

### DISCUSSION THREAD

**Discussion by dizmoduck** [OpenTracksApp/OpenTracks#1509] [Ideas]
On the aggregated stats tab and on the filter tab, it would be cool to get the combined time and km for the selected kelgories?  

Perhaps also a page with statistics figures per week, month and year

Sorry I'm not a coder, but I have a lots of ideas and I will help to test it, just like with the color of speed on the dashboard.

I have just under 1000 trackings from 2010 to today 

 Keep op the good work 

![Screenshot_20230326-182649](https://user-images.githubusercontent.com/8668926/227791028-f942d5c3-7086-42ff-ac55-1ab538c9f651.png)
![Screenshot_20230326-182629](https://user-images.githubusercontent.com/8668926/227791034-f7b1def6-44c8-4816-8329-9d08946b9955.png)

**Comment by dennisguse** [comment]
I guess, I didn't understand what you need...

**Reply by dizmoduck** [reply]
sorry I wasn't clear enough, I'm thinking of getting a combined time and km of the selected kelgories?

As it is now you only get the total stats for each kelgories?  I want to get the total sum of selected kelgories e.g. if I selected cycling, walking and running then the combined sum of km and moving time of the 3 selected kelgories is displayed? preferably per week, month and year on there own tap

With the option to choose from and to date as you can now.

Hope it makes more sense!

**Reply by dennisguse** [reply]
Can you open an issue for this?

**Reply by dizmoduck** [reply]
Done
